File: 프로그래머스/lv1/12977. 소수 만들기/소수 만들기.java
import java.util.Arrays;
class Solution {
    public int solution(int[] nums) {
        int answer = 0;
        
        for(int i = 0; i < nums.length - 2; i ++){
            for(int j = i + 1; j < nums.length - 1; j ++){
                for(int k = j + 1; k < nums.length; k ++ ){
                    if(isPrime(nums[i] + nums[j] + nums[k])){
                        answer++;  
                    } 
                }
            }
        }
        return answer;
    }
    public Boolean isPrime(int num){
        int cnt = 0;
        for(int i = 1; i <= (int)Math.sqrt(num); i++){
            if(num % i == 0) {
                cnt++;
            }
        }
        return cnt == 1;
    }
}

File: 프로그래머스/lv1/12977. 소수 만들기/소수 만들기.py
import math


def solution(nums):
    answer = 0
    num = []
    for i in range (len(nums)):
        for j in range (i+1,len(nums)):
            for k in range (j+1,len(nums)):
                num.append(nums[i] + nums[j] + nums[k])
    for i in num:
        for k in range (2, int(math.sqrt(i)) + 1):
            if bool(i % k == 0):
                break
        else:
            answer += 1
    return answer